Config/config.php

```diff
@@ -41,6 +42,29 @@
     ],
 
     'services' => [
+        'integrations' => [
+            'mautic.integration.triggerdialog' => [
+                'class'     => TriggerdialogIntegration::class,
+                'arguments' => [
```

Comment:
Are all these arguments really necessary?

Reply:
no, I removed the whole integration config, which was unnecessary